src/graphql/resolvers/index.js

```diff
@@ -271,14 +274,7 @@ module.exports = function Resolvers (ssb) {
 
         return new Promise((resolve, reject) => {
           getProfile(alias.userId)
-            .then((profile) => {
-              if (!profile) return resolve(null)
-
-              // the alias gets passed on the the child resolver Profile.ssbURI
-              profile.alias = alias
-
-              resolve(profile)
-            })
+            .then(resolve)
```

Comment:
Reviewing on phone but I think we don't need this new Promise any more right? Doesn't get Profile return a promise?